pandas/io/parsers.py

```diff
@@ -2194,16 +2194,17 @@ def _handle_usecols(self, columns, usecols_key):
         usecols_key is used if there are string usecols.
         """
         if self.usecols is not None:
-            if any([isinstance(u, string_types) for u in self.usecols]):
+            if any([isinstance(usecol, string_types)
+                    for usecol in self.usecols]):
```

Comment:
@jorisvandenbossche : The `u` shadows the `u` import from `pandas.compat`.  I don't see any problem with making this expanded as `usecol`.